packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -2349,21 +2349,51 @@ class TestWidgetInspectorService extends Object with WidgetInspectorService {
           context: ErrorDescription('during layout'),
           exception: StackTrace.current,
         ));
-      } finally {
-        FlutterError.onError = oldHandler;
-      }
 
-      // validate that we received an error
-      flutterErrorEvents = service.getEventsDispatched('Flutter.Error');
-      expect(flutterErrorEvents, hasLength(1));
+        // validate that we received an error
+        flutterErrorEvents = service.getEventsDispatched('Flutter.Error');
+        expect(flutterErrorEvents, hasLength(1));
+
+        // validate the error contents
+        Map<Object, Object> error = flutterErrorEvents.first;
+        expect(error['description'], 'Exception caught by rendering library');
+        expect(error['children'], isEmpty);
+
+        // validate that we received an error count
+        expect(error['errorsSinceReload'], 0);
+
+        // send a second error
+        FlutterError.reportError(FlutterErrorDetailsForRendering(
+          library: 'rendering library',
+          context: ErrorDescription('also during layout'),
+          exception: StackTrace.current,
+        ));
+
+        // validate that the error count increased
```

Comment:
Nit: make all these comments start with a capital letter and end with a period to match the rest of the file.

Reply:
done